lambdas/shared/tests/test_decorator.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
worth a quick `pylint` on this file. i'm not huge on docstrings for tests but might as well set the bar :P